### PR TITLE
add Ross temperature model

### PR DESCRIPTION
**Pull request by cwhanse**
- [x] I am familiar with the [contributing guidelines](https://pvlib-python.readthedocs.io/en/latest/contributing.html)
 - [x] Tests added
 - [x] Updates entries to [`docs/sphinx/source/api.rst`](https://github.com/pvlib/pvlib-python/blob/master/docs/sphinx/source/api.rst) for API changes.
 - [x] Adds description and name entries in the appropriate "what's new" file in [`docs/sphinx/source/whatsnew`](https://github.com/pvlib/pvlib-python/tree/master/docs/sphinx/source/whatsnew) for all changes. Includes link to the GitHub Issue with `` :issue:`num` `` or this Pull Request with `` :pull:`num` ``. Includes contributor name and/or GitHub username (link with `` :ghuser:`user` ``).
 - [x] New code is fully documented. Includes [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) compliant docstrings, examples, and comments where necessary.
 - [x] Pull request is nearly complete and ready for detailed review.
 - [x] Maintainer: Appropriate GitHub Labels and Milestone are assigned to the Pull Request and linked Issue.

A simple cell temperature model using only NOCT as a parameter.

### CODE REVIEW

File: pvlib/temperature.py [hunk 1]
```diff
@@ -443,6 +444,50 @@ def faiman(poa_global, temp_air, wind_speed=1.0, u0=25.0, u1=6.84):
     return temp_air + temp_difference
 
 
+def ross(poa_global, temp_air, noct):
+    r'''
+    Calculate cell temperature using the Ross model.
+
+    The Ross model [1]_ assumes that cell temperature is proportional to the
+    plane of array irradiance, and ignores the effects of wind.
```
Comment (kanderso-nrel) [inline pvlib/temperature.py:452]: Strictly speaking, it is the difference between ambient and cell temperature that is proportional to irradiance, not the cell temperature itself.

File: pvlib/temperature.py [hunk 2]
```diff
@@ -443,6 +444,50 @@ def faiman(poa_global, temp_air, wind_speed=1.0, u0=25.0, u1=6.84):
...
+    Parameters
+    ----------
+    poa_global : numeric
+        Total incident irradiance [W/m^2].
+
+    temp_air : numeric
+        Ambient dry bulb temperature [C].
+
+    noct : numeric
+        Nominal operating cell temperature, determiend at conditions of
```
Comment (kanderso-nrel) [inline pvlib/temperature.py:463]: "determined" typo

File: pvlib/temperature.py [hunk 3]
```diff
@@ -443,6 +444,50 @@ def faiman(poa_global, temp_air, wind_speed=1.0, u0=25.0, u1=6.84):
...
+    Returns
+    -------
+    numeric, values in degrees Celsius
+
+    Notes
+    -----
+    The Ross model for cell temperature :math:`T_{C}` is given in [1]_ as
+
+    .. math::
+       :label: pvsyst
```
Comment (kanderso-nrel) [inline pvlib/temperature.py:475]: "pvsyst" copy/paste artifact

File: pvlib/temperature.py [hunk 4]
```diff
@@ -443,6 +444,50 @@ def faiman(poa_global, temp_air, wind_speed=1.0, u0=25.0, u1=6.84):
...
+
+        T_{C} = T_{a} + \frac{NOCT - 20}{80} S
+
+    where :math:`S` is the plane of array irradiance in mW/{cm}^2.
```
Comment (kanderso-nrel) [inline pvlib/temperature.py:479]: Are these braces intended?  Or maybe the unit should be wrapped in a `:math:` tag?

File: pvlib/temperature.py [hunk 5]
```diff
@@ -443,6 +444,50 @@ def faiman(poa_global, temp_air, wind_speed=1.0, u0=25.0, u1=6.84):
...
+    ----------
+    poa_global : numeric
+        Total incident irradiance [W/m^2].
+
+    temp_air : numeric
+        Ambient dry bulb temperature [C].
+
+    noct : numeric
+        Nominal operating cell temperature, determiend at conditions of
+        800 W/m^2 irradiance, 20C ambient air temperature and 1 m/s wind.
```
Comment (kanderso-nrel) [inline pvlib/temperature.py:464]: perhaps `[C]` at the end?

File: docs/sphinx/source/whatsnew/v0.8.0.rst [hunk 6]
```diff
@@ -96,6 +96,8 @@ Enhancements
 * Add :py:func:`pvlib.pvsystem.combine_loss_factors` as general purpose
   function to combine loss factors with a common index.
   Partialy addresses :issue:`988`. Contributed by Brock Taute :ghuser:`btaute`
+* Added :py:func:`pvlib.temperature.ross` for cell temperature modeling using
+  only NOCT. (:pull:`1045`)
```
Comment (kanderso-nrel) [inline docs/sphinx/source/whatsnew/v0.8.0.rst:100]: Move to the 0.8.1 whatsnew file

**Review by kanderso-nrel — APPROVED** [review]
LGTM

File: docs/sphinx/source/api.rst [hunk 7]
```diff
@@ -236,7 +236,7 @@ PV temperature models
    temperature.pvsyst_cell
    temperature.faiman
    temperature.fuentes
-   pvsystem.PVSystem.sapm_celltemp
```
Comment (wholmgren) [inline docs/sphinx/source/api.rst:239]: https://github.com/pvlib/pvlib-python/pull/1054#discussion_r484982981 and #1055 

I'd leave it here for now.

File: docs/sphinx/source/whatsnew/v0.8.1.rst [hunk 8]
```diff
@@ -13,6 +13,8 @@ Deprecations
 
 Enhancements
 ~~~~~~~~~~~~
+* Added :py:func:`pvlib.temperature.ross` for cell temperature modeling using
+  only NOCT. (:pull:`1045`)
```
Comment (wholmgren) [inline docs/sphinx/source/whatsnew/v0.8.1.rst:17]: and add yourself to contributors below. I also forgot to add myself for the expanded asv tests.

File: pvlib/temperature.py [hunk 9]
```diff
@@ -443,6 +444,50 @@ def faiman(poa_global, temp_air, wind_speed=1.0, u0=25.0, u1=6.84):
     return temp_air + temp_difference
 
 
+def ross(poa_global, temp_air, noct):
+    r'''
+    Calculate cell temperature using the Ross model.
+
+    The Ross model [1]_ assumes that difference betwen cell temperature
```
Comment (wholmgren) [inline pvlib/temperature.py:451]:
```suggestion
    The Ross model [1]_ assumes the difference between cell temperature
```

File: pvlib/temperature.py [hunk 10]
```diff
@@ -443,6 +444,50 @@ def faiman(poa_global, temp_air, wind_speed=1.0, u0=25.0, u1=6.84):
...
+    ----------
+    poa_global : numeric
+        Total incident irradiance [W/m^2].
+
+    temp_air : numeric
+        Ambient dry bulb temperature [C].
+
+    noct : numeric
+        Nominal operating cell temperature, determined at conditions of
+        800 W/m^2 irradiance, 20C ambient air temperature and 1 m/s wind. [C]
```
Comment (wholmgren) [inline pvlib/temperature.py:465]:
```suggestion
    noct : numeric
        Nominal operating cell temperature [C], determined at conditions of
        800 W/m^2 irradiance, 20 C ambient air temperature and 1 m/s wind.
```

File: pvlib/temperature.py [hunk 11]
```diff
@@ -443,6 +444,50 @@ def faiman(poa_global, temp_air, wind_speed=1.0, u0=25.0, u1=6.84):
...
+
+    Returns
+    -------
+    numeric, values in degrees Celsius
```
Comment (wholmgren) [inline pvlib/temperature.py:469]:
```suggestion
    cell_temperature : numeric
        Cell temperature in degrees Celsius.
```

**Review by wholmgren — APPROVED** [review]
Looks good overall. Minor stuff below.

File: pvlib/temperature.py [hunk 12]
```diff
@@ -443,6 +444,50 @@ def faiman(poa_global, temp_air, wind_speed=1.0, u0=25.0, u1=6.84):
     return temp_air + temp_difference
 
 
+def ross(poa_global, temp_air, noct):
+    r'''
+    Calculate cell temperature using the Ross model.
+
+    The Ross model [1]_ assumes that difference betwen cell temperature
+    and ambient temperature is proportional to the plane of array irradiance,
+    and ignores the effects of wind.
```
Comment (wholmgren) [inline pvlib/temperature.py:453]: @adriesse recently reminded us of #433. In that spirit, I suggest we encourage new functions to include documentation about any additional assumptions that went into the model development e.g. hourly data or zenith restrictions.

File: docs/sphinx/source/whatsnew/v0.8.1.rst (hunk 8 above)
Comment (cwhanse) [reply docs/sphinx/source/whatsnew/v0.8.1.rst:17]: I'll add

File: pvlib/temperature.py (hunk 12 above)
Comment (cwhanse) [reply pvlib/temperature.py:453]: Adding the implicit assumption of steady conditions.

**Comment by adriesse** [comment]
Should this really be called a "Ross" model?  It's just basic heat loss and NOCT conditions:

```
total_loss_factor  = 800 / (noct - 20)
heat_input = poa_global
temp_difference = heat_input / total_loss_factor
return temp_air + temp_difference
```
I wouldn't say this approach ignores the effect of wind but rather that it tacitly assumes a wind speed of 1 m/s.

**Comment by cwhanse** [comment]
> Should this really be called a "Ross" model? It's just basic heat loss and NOCT conditions:

Agree that's what it is, but it needs a name, and others have cited it as Ross' model (author of the reference).

> I wouldn't say this approach ignores the effect of wind but rather that it tacitly assumes a wind speed of 1 m/s.

good point.

File: pvlib/temperature.py [hunk 13]
```diff
@@ -443,6 +444,53 @@ def faiman(poa_global, temp_air, wind_speed=1.0, u0=25.0, u1=6.84):
     return temp_air + temp_difference
 
 
+def ross(poa_global, temp_air, noct):
+    r'''
+    Calculate cell temperature using the Ross model.
+
+    The Ross model [1]_ assumes the difference between cell temperature
+    and ambient temperature is proportional to the plane of array irradiance,
+    and ignores the effects of wind. The model implicitly assumes steady or
```
Comment (cwhanse) [inline pvlib/temperature.py:453]:
```suggestion
    and assumes wind speed of 1 m/s. The model implicitly assumes steady or
```

**Review by cwhanse — COMMENTED** [review]
change docstring

**Comment by wholmgren** [comment]
thanks @cwhanse!

**Pull request merged**